elixir/apps/domain/priv/repo/migrations/20250530194329_update_verified_by_constraint.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Domain.Repo.Migrations.UpdateVerifiedByConstraint do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+    ALTER TABLE clients
+    DROP CONSTRAINT IF EXISTS verified_fields_set
+    """)
+
+    create(
+      constraint(:clients, :verified_fields_set,
+        check: """
+        (
+          verified_at IS NULL
+          AND (verified_by IS NULL AND verified_by_subject IS NULL)
```

Comment:
(nit) extra parens aren't required?

Reply:
Yeah, good point.  I pulled from a prior migration, but I'll remove it.